.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,15 @@
+name: "Chromatic"
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: |
+        yarn && yarn build
+    - uses: chromaui/action@v1
+      with: 
+        projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+        token: ${{ secrets.GITHUB_TOKEN }}
+        buildScriptName: storybook:chromatic
```

Comment:
I know this is a draft so I'll go away but I was curious 🤣  

I'm so excited about this!!! Flee from travis!!!! 🏃‍♂️ 🏃 🏃‍♀️

Reply:
Possibly dumb question - will this work for forks? I can't see why not - the only thing I would be worried about is if the secret gets passed

Reply:
Not at all a dumb question. You're correct. Yikes: https://www.chromatic.com/docs/ci#run-chromatic-on-external-forks-of-open-source-projects

Is it a good solution to expect fork authors to ask repo maintainers to trigger Chromatic 🤷‍♂️ ?

Reply:
Also, I think @droshev has set up a free open-source plan for us. If that's the case, we have unlimited snapshots, and sharing the token should be fine.